sources/models/src/aws-ecs-1/override-defaults.toml

```diff
@@ -12,3 +12,4 @@ affected-services = ["ecs"]
 
 [settings.ecs]
 allow-privileged-containers = false
+logging-drivers = ["json-file", "journald", "awslogs", "none"]
```

Comment:
I'm wondering if enabling `journald` should be listed here if this is a suggested production setting: journal access is limited without the use of the admin container. I suspect we'll see many more users running with log-shipper drivers or transports. Thoughts?

Reply:
> I'm wondering if `journald` should be listed here if this is a suggested production setting: journal access is limited without the use of the admin container.

This is true for `json-file` as well, though you can get access to that by mounting the Docker socket.

> I suspect we'll see many more users running with log-shipper drivers or transports.

Agreed.

> Thoughts?

I don't have a strong opinion on it; I lean toward leaving it in but am happy to remove if you think that's better.

Reply:
> This is true for `json-file` as well, though you can get access to that by mounting the Docker socket.

This pattern, in my experience, appears to be understood and widely accepted - even if it blurs security boundaries in the process. I've seen logging tools (such as [elastic's `beat` docker log input](https://github.com/elastic/beats/blob/aa9c1eab64686203b9f1e606f304d8f35613d976/filebeat/docs/inputs/input-docker.asciidoc#docker-input)) that use the buffered stream from `json-file` that then ship off or handle events by listening in on the log using `json-file` as the driver.

I'm in favor of removing `journald` from the default set. However, `json-file` should remain in the set; `json-file` is *docker*'s default log driver and not including it would prevent tasks from scheduling even if they don't have LogConfiguration set.

Reply:
> I'm in favor of removing `journald` from the default set.

Done

> However, `json-file` should remain in the set; `json-file` is _docker_'s default log driver and not including it would prevent tasks from scheduling even if they don't have LogConfiguration set.

This isn't quite accurate.  If LogConfiguration isn't set, no explicit or implicit placement constraints are applied; the task can be placed anywhere in the cluster without respect to log drivers and it will use whatever the default driver is on the instance where it is placed (`json-file` is the default, but that can be configured in Docker's config file).